docs/embedding/embedded-analytics-js.md

```diff
@@ -348,3 +351,20 @@ To render a question (chart).
 - `initial-sql-parameters` - default value for SQL parameters, only applicable to native SQL questions, like `{ "productId": "42" }`
 - `is-save-enabled` (default is false)
 - `target-collection` - this is to enforce saving into a particular collection. Values: regular ID, entity ID, `"personal”`, `"root”`
+
+- ### Browser
+
+To render a collection browser so people can navigate a collection and open dashboards or questions.
+
+ ```html
+<metabase-browser initial-collection="14" read-only="false"></metabase-browser>
+```
+
+**Required parameters:**
+
+- `initial-collection` - This can be a collection ID or "root". Use a collection ID (e.g. 14) to start in a specific collection. Use "root" to start at the top level with all collections.
+
+**Optional parameters:**
+
+- `read-only` (default is true) – if true, people can only view and explore. If false, they can create and edit dashboards and questions in the collection.
```

Comment:
```suggestion
- `read-only` (default is true) – if true, people can only view and explore. If false, they can create and edit items in the collection.
```

To anticipate things like documents.

Also, explain what "view and explore" means.